docs/FAQ/Oracles.md

```diff
@@ -0,0 +1,18 @@
+## I am currently using Chainlink. How difficult is it to change to Chronicle?
+If you are using Chainlink, all you need to do is to change the address of your oracle to the Chronicle equivalent. This is because Scribe is partially Chainlink compatible, implementing the most widely, and not deprecated, used functions of the `IChainlinkAggregatorV3` interface such as `latestRoundData()`, `decimals()`, and `latestAnswer()`.
```

Comment:
This is a tricky one... Chainlink uses different decimals for different oracles. We always use 18 decimals. If they implemented the oracle correctly, they alway check the decimals and scale them up/down appropriately.

However, I expect many projects to have taken shortcuts... Maybe lets use a less strong phrasing? We know about this issue but didn't proactively work on it yet. However, a proxy contract is available (but nowhere deployed yet), see https://github.com/chronicleprotocol/scribe-chainlinklike.

Maybe we can do something like: "Please do integration tests. Feel free to reach out if there are issues"?